insights/client/connection.py

```diff
@@ -871,89 +899,94 @@ def upload_archive(self, data_collected, content_type, duration=None):
         }
         logger.debug("Uploading %s to %s", data_collected, upload_url)
 
-        logger.log(NETWORK, "POST %s", upload_url)
-        upload = self.session.post(upload_url, files=files, headers={})
-
-        logger.log(NETWORK, "Upload status: %s %s %s",
-                     upload.status_code, upload.reason, upload.text)
+        upload = self.post(upload_url, files=files, headers={})
```

Comment:
What’s the `headers={}` there used for? I see it has not been added in this pull request, but still.

```suggestion
        upload = self.post(upload_url, files=files)
```